manifests/rule.pp

```diff
@@ -207,15 +211,19 @@
     fail("Logrotate::Rule[${rulename}]: create_mode requires create")
   }
 
-  if $su_owner and !defined('$su_group') {
-    $_su_owner = $su_owner
-    $_su_group = 'root'
-  } elsif !defined('$su_owner') and $su_group {
-    $_su_owner = 'root'
-    $_su_group = $su_group
-  } else {
-    $_su_owner = $su_owner
-    $_su_group = $su_group
+  $_su = $su ? {
```

Comment:
yey selector operator \o/

Reply:
I assume you are not sarcastic and agree this is indied better... :)